datadog_test.go

```diff
@@ -216,7 +225,10 @@ func TestSumData(t *testing.T) {
 
 func TestLastValueData(t *testing.T) {
 	reportPeriod := time.Millisecond
-	exporter := testExporter(Options{})
+	exporter, err := testExporter(Options{})
+	if err != nil {
+		t.Error(err)
```

Comment:
This `t.Error` and all the other ones should be replaced by `t.Fatal`. Is there any point to continue this test if we get an error?